src/tests/ftest/checksum/csum_error_logging.py

```diff
@@ -97,6 +97,8 @@ def test_csum_error_logging(self):
         dev_id = self.get_nvme_device_id()
         self.log.info("%s", dev_id)
         csum = self.get_checksum_error_value(dev_id)
+        ##DH++
+        self.dmg.copy_certificates(get_log_file("daosCA/certs"), [self.host])
```

Comment:
(pylint-undefined-variable) Undefined variable 'get_log_file'
(pylint-no-member) Instance of 'CSumErrorLog' has no 'host' member